app/controllers/application_controller.rb

```diff
@@ -1,2 +1,8 @@
 class ApplicationController < ActionController::Base
+  # Only the autenticated users are able to navigate  in the app
+  before_action :authenticate_user!
```

Comment:
@sansuaza what a **before_action** does?

Reply:
before_actions is call back, its function is to stop the exection of the request to run something before that is necessary to give an answer.

Reply:
@sansuaza not really sure about the .. **"stop the execution"** are you really sure about it?